### PR TITLE
mod(n, range)

### PR DESCRIPTION
**Pull request by mcabbott**
There are lots of recent small additions to Julia I’d like to be able to use on 1.2. Tell me if I’m doing this right, and then I’ll do a few more?

### CODE REVIEW

File: src/Compat.jl [hunk 1]
```diff
@@ -1866,6 +1866,12 @@ if v"0.7.0" <= VERSION < v"1.1.0-DEV.594"
     Base.merge(a::NamedTuple) = a
 end
 
+# https://github.com/JuliaLang/julia/pull/32628
+if v"0.7.0" <= VERSION < v"1.3.0-DEV.546"
```
Comment (martinholters) [inline src/Compat.jl]: I get a different version for the merge commit:
```
$ ./contrib/commit-name.sh 21823899cc1724871a8e679287a94bb4acee72d7
1.3.0-alpha.8
```
Can you double-check?

Comment (mcabbott) [reply src/Compat.jl]: That’s better, It looks like I picked the last of my commits:
```
$ contrib/commit-name.sh 21823899cc1724871a8e679287a94bb4acee72d7
1.3.0-alpha.8
$ contrib/commit-name.sh f93604aa24ade37fa4ca7a1c0446894dd27543ee
1.3.0-DEV.546
```

**Comment by martinholters** [comment]
Thanks for the contribution!

The list of links at the end of the README needs to be updated, but we can do that in a seperate PR. Just shouldn't forget before tagging.

**Comment by mcabbott** [comment]
Ah I didn’t scroll down that far. What’s the list for? (I also see that I failed to make the README line link to the issue.)

I find the Readme pretty confusing BTW, wasn’t sure whether to add at the top or bottom, apologies if I made it worse. I guess #652 will help a lot.

**Pull request merged**